src/main/interface/cli.c

```diff
@@ -439,13 +443,21 @@ static void printValuePointer(const clivalue_t *var, const void *valuePointer, b
 
         switch (var->type & VALUE_MODE_MASK) {
         case MODE_DIRECT:
-            cliPrintf("%d", value);
-            if (full) {
-                cliPrintf(" %d %d", var->config.minmax.min, var->config.minmax.max);
+            if ((value < var->config.minmax.min) || (value > var->config.minmax.max)) {
+                cliPrintCorruptMessage();
```

Comment:
With direct value, it may be better to print it and add out of range (or corrupt) comment

Reply:
Yes please, we don't need entropy increase :)

Reply:
Good idea. I was originally going to do this but didn't because I wanted to make sure the `diff` output wasn't "paste-able" to propagate the invalid value.  But that shouldn't be a problem because the validation should prevent it.  No reason not to display the invalid index for lookup values either I think.  Can only help with troubleshooting or trying to figure out how the corrupted values got there.  I'll update.